Sources/Plugins/AttachmentManager/AttachmentManager.swift

```diff
@@ -191,8 +205,28 @@ extension AttachmentManager: UICollectionViewDataSource, UICollectionViewDelegat
                 cell.imageView.tintColor = tintColor
                 cell.deleteButton.backgroundColor = tintColor
                 return cell
+                
+            case .video(let video):
+                guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: VideoAttachmentCell.reuseIdentifier, for: indexPath) as? VideoAttachmentCell else {
+                    fatalError()
+                }
+                DispatchQueue.main.async {
+                    cell.attachment = attachment
+                    cell.indexPath = indexPath
+                    cell.manager = self
+                    cell.imageView.image = video.thumbnailImage ?? UIImage()
+                    if let duration = video.duration {
+                        let durationString = self.videoDurationToString(seconds: Int(duration))
+                        cell.durationLabel.text = durationString
+                    } else {
+                        cell.durationLabel.isHidden = true
+                    }
+                    cell.imageView.tintColor = self.tintColor
+                    cell.deleteButton.backgroundColor = self.tintColor
+                }
+                return cell
             default:
-                return collectionView.dequeueReusableCell(withReuseIdentifier: AttachmentCell.reuseIdentifier, for: indexPath) as! AttachmentCell
+                return collectionView.dequeueReusableCell(withReuseIdentifier: ImageAttachmentCell.reuseIdentifier, for: indexPath) as! ImageAttachmentCell
```

Comment:
This doesn't feel like a correct change - this type should be handled via image case, but generally fallback to universal cell is correct?